config/v1/types_image_content_policy.go

```diff
@@ -42,8 +42,7 @@ type ImageContentPolicySpec struct {
 	// mirrors will be used in the order `a, b, c, d, e`.  If the orders of mirror entries conflict
 	// (e.g. `a, b` vs. `b, a`) the configuration is not rejected but the resulting order is unspecified.
 	// +optional
-	// +listType=map
-	// +listMapKey=source
+	// +listType=atomic
```

Comment:
> … or maybe we can just remove the `allowMirrorByTags` option entirely, and hard-code it to `true`?

@mtrmac  This can simplify the implementation. But I have no idea about the use cases about allowMirrorByTags false. Is it possible that many features requests appear that require `allowMirrorByTags` configurable after we hardcode it to true?